bindings/python/test/time/test_date_time.py

```
@@ -1,6 +1,8 @@
 # Apache License 2.0
 
-import datetime
+from typing import cast
```

Ruff (0.8.2)

`typing.cast` imported but unused

Remove unused import: `typing.cast`

(F401)